client/src/components/common/ProfileCard/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+import handymanImg from '../../../assets/handyman.png';
+import Button from '../Button';
+
+const ProfileCard = ({ username, service, country, city, hourRate, bio }) => {
+  return (
+    <div className="card">
+      <div className="card__header">
+        <img className="card__header--img" src={handymanImg} alt="handyman" />
+        <div className="card__header--info">
+          <h3 className>{username}</h3>
+          <h4>{service}</h4>
+          <h4>
+            {country}, {city}
+          </h4>
+          <h4>{hourRate}$/hr</h4>
+          <div className="btn-container">
+            <Button
+              className="message"
+              children="massege"
+              buttonFunction={undefined}
+            />
+            <Button
+              className="hire"
+              children="hire"
+              buttonFunction={undefined}
```

Comment:
I prefer to take the function from props and pass it here because we use this function in different components. so we can raise an event and handle it in the parent component